controllers/provisioning_controller.go

```diff
@@ -112,21 +108,6 @@ func (r *ProvisioningReconciler) readProvisioningCR(req ctrl.Request) (*metal3io
 func (r *ProvisioningReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	//log := r.Log.WithValues("provisioning", req.NamespacedName)
 
-	enabled, err := r.isEnabled()
```

Comment:
On a second thought, what about keeping just a safety check to prevent the reconcile loop running, in case the Provisioning resource will be deployed by mistake?

Reply:
I am not sure if we can claim to be "Available=True" without the reconcile loop running. Let me look into this some more.

Reply:
I misread your comment earlier. I think we need to add this check back not because the controller now watches the CO resource too. (https://github.com/openshift/cluster-baremetal-operator/blob/master/controllers/provisioning_controller.go#L239)